_posts/en/posts/schnorr-sigagg.md

```diff
@@ -0,0 +1,84 @@
+---
+type: posts
+layout: post
+lang: en
+name: schnorr-sigagg
+id: en-schnorr-sigagg
+title: Technology roadmap: Schnorr signatures & signature aggregation
+permalink: /en/2017/03/22/schnorr-sigagg/
+version: 1
+excerpt: Status and explanation of Schnorr signatures and signature aggregation
+---
+
+## Schnorr Signatures
+
+The replacement of Bitcoin’s digital signature algorithm (ECSDA) for the more efficient Schnorr algorithm has long been at the top of the bucket list for many Bitcoin developers. A simple algorithm leveraging elliptic curves cryptography, Schnorr enables several improvements over the existing scheme all while preserving all of the features and security assumptions of ECSDA.
+
+Notably, Schnorr signatures support “native multisig” which enables the aggregation of multiple signatures into a single one valid for the sum of the keys of their respective inputs. This functionality offers three important benefits: 
+
+Constant-size signatures irrespective of the number of participants in the multisig setup. A 50-of-50 transaction is effectively the same size as one that uses a single public key and signature. For this reason, the performance of such schemes is significantly improved by removing the original requirement of validating every signature individually. Additionally, the verification of Schnorr signatures is slightly faster than that of ECSDA.
+
+
+The diminished size of data to be validated and transmitted across the network also translates in interesting capacity gains. Considering the historical growth in the number of multisig transactions displayed below, the potential to reduce the size of these transactions is an enticing addition to existing scaling efforts. We should expect this trend to continue with the emergence and further adoption of payment channels.
+
+
+From a privacy standpoint, Schnorr allows the entire policy of the multisig to be obscured and indistinguishable from a conventional single pubkey. In a threshold setup, it also becomes impossible for participants to reveal which of them authorized, or not, a transaction. 
+
+
+<p align="center">
+  <img src="https://cdn-images-1.medium.com/max/800/1*9_03N_A0OlLkcJpCGq--Qw.png">
```

Comment:
I suggest that this image and the imgur one below be added in this pull request and linked to locally.  This (1) means the page isn't dependent on external resources, so it's less likely to quietly break on us and (2) means that Medium and Imgur don't get to see the IP address and otherwise track everyone who visits this page on BitcoinCore.org.